src/index.html

```diff
@@ -7,7 +7,6 @@
     <title>LeaderBoard</title>
   </head>
   <body>
-    <h1>LeaderBoard</h1>
```

Comment:
- [ ] As stated the project requirement Kindly  keep the general layout of the wireframe, this is the only mandatory requirement. So Kindly add "LeaderBoard " header Just like the wireframe.

Your Project Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/209161906-22a0a5a4-5d8d-4dfe-a4f8-ca90660839db.PNG)

The Leaderboard: Final Touches wireframe Look likes this  : 

![Capture](https://user-images.githubusercontent.com/93598189/209162206-87bda78d-b24c-4895-bff2-01f254a99402.PNG)